spec/interactors/users/check_credentials_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "rails_helper"
+
+describe Users::CheckCredentials do # rubocop:disable Metrics/BlockLength
+  describe ".call" do # rubocop:disable Metrics/BlockLength
+    let!(:user) do
+      {
+        first_name: "test",
+        last_name: "test",
+        email: "person@admin.ru",
+        password: "123456"
+      }
+    end
+
+    context "when credentials are valid" do
+      let!(:spec_credentials) do
+        {
+          first_name: "test",
+          last_name: "test",
+          email: "person@admin.ru",
+          password: "123456"
+        }
+      end
+
+      let!(:interactor) do
+        described_class
+          .new(credentials: spec_credentials)
+      end
+
+      it "and user params equals to context.credentials" do
+        interactor.run
+
+        expect(interactor.context.credentials).to eq(user)
```

Comment:
If it is user params, name it `user_params`, not `user`

Reply:
Yes, sure. But I will use user for this test. I need to change description of test method